crates/router/src/connector/authorizedotnet/transformers.rs

```diff
@@ -341,12 +368,27 @@ impl TryFrom<&AuthorizedotnetRouterData<&types::PaymentsAuthorizeRouterData>>
             }),
             None => None,
         };
-
+        let bill_to = match item.router_data.get_billing() {
```

Comment:
If you don't want to throw error please then use item.router_data.get_billing_address_details_as_optional which return option and instead of match statement you can use map

Reply:
Done